### ISSUE TITLE
안녕하세요 공부 순서에 대해서 질문드리고 싶습니다!

### ISSUE THREAD

**Issue by jun-hyun-web** [codingeverybody/codingyahac#884]
복수 전공을 시작한지 현재 6개월 차이고 본전공보다 솦대공부가 훨씬 적성에 맞고 공부하는 시간이 즐거운 학생입니다! 다름이 아니라 공부 순서에 대해서 여쭤보고 싶은것이 있어서 이렇게 문의드려요!
 
먼저 저번학기에는 단순히 c언어를 위주로 하는 알고리즘 수업이나 어셈블리코드 같은 것을 배웠습니다! 그리고 방학이 찾아와서 제가 혼자 프로젝트를 한번 진행해보고싶다는 욕심이 생겨서 웹-어플을 만들어보려고 공부를 하는중입니다! 현재 유튜브에서 생활코딩의 html css javascript이렇게 세개를 들은 상태입니다! nodejs apache mysql정도를 공부하라고 친구에게 들었는데 이것들의 연관성을 잘 모르니 공부를 해야하는 순서도 잘모르겠더라구요.. 기본 베이스는 아까말한 html css javascript밖에 없는 상황입니다 생활코딩의 어떤 재생목록을 먼저듣고 어디까지 들은 후에 간단히 제가 스스로 어플을 만들수있을까요? 간단한 어플 만들고 싶어요 ㅎㅎ 예를들면 술집검색어플 같은거요!

**Comment by o2-basta** [comment]
네 node, mysql, apache를 배우시면 검색을 해서 원하는 것들을 찾아주거나 하는 부분은 만들 수 있어요 ㅎㅎ

다만, 현재 배우신 부분과 연결지어서 배움을 확장하는데 있어서 막막하게 다가오는 부분이 클거 같아요

배우긴 해야 하는데, 접해본적도 없는 부분이라서 선뜻 어떻게 해야할지 모를 수 있어요. 

우선 node라는 것을 사용하신다면, 서버에서 웹페이지를 보내주는 부분을 node로 배우신다면 이미 알고계신 javascript를 통해서 웹페이지를 제공하는 기능적인 부분을 해보시는것을 추천드립니다. node만 이해해도 웹페이지를 보여주는데는 문제가 없어요 


그러고 나면 데이터베이스를 공부하시면서 검색대상들을 저장해보고 꺼내보는걸 해보시면 감을 잡을 수 있을거 같아요

그리고, apache 같은것은 웹페이지를 전문적으로 보여주는 웹서버이지만, node를 개발하면서 기본적인 웹페이지는 서비스해볼 수 있으니, 원하는것을 만드는데 **필수적**인것은 node라는것과 데이터베이스일것이라고 생각합니다. 

모르는 부분을 접하는것이라서 어색한것 뿐이니, 금방 익숙해질 수 있어요!
